pallets/identity/src/lib.rs

```diff
@@ -528,6 +527,44 @@ decl_module! {
         pub fn rotate_primary_key_to_secondary(origin, auth_id:u64, optional_cdd_auth_id: Option<u64>) -> DispatchResult {
             Self::base_rotate_primary_key_to_secondary(origin, auth_id, optional_cdd_auth_id)
         }
+
+        /// It adds secondary keys to target identity `id`.
+        /// Keys are directly added to identity because each of them has an authorization.
+        ///
+        /// Arguments:
+        ///     - `origin` Primary key of `id` identity.
+        ///     - `id` Identity where new secondary keys will be added.
+        ///     - `additional_keys` New secondary items (and their authorization data) to add to target
+        ///     identity.
+        ///
+        /// Failure
+        ///     - It can only called by primary key owner.
+        ///     - Keys should be able to linked to any identity.
```

Comment:
```suggestion
        /// Adds secondary keys to target identity `id`.
        ///
        /// Keys are directly added to identity because each of them has an authorization.
        ///
        /// # Arguments:
        ///     - `origin` which must be the primary key of the identity `id`.
        ///     - `id` to which new secondary keys will be added.
        ///     - `additional_keys` which includes secondary keys,
        ///        coupled with authorization data, to add to target identity.
        ///
        /// # Errors
        ///     - Can only called by primary key owner.
        ///     - Keys should be able to linked to any identity.
```